Library/Homebrew/cmd/list.rb

```diff
@@ -76,7 +76,8 @@ def list
     lib/ruby/gems/[12].*
     lib/ruby/site_ruby/[12].*
     lib/ruby/vendor_ruby/[12].*
-    manpages/brew.1
+    manpages/man1/brew.1
+    manpages/man1/brew-cask.1
```

Comment:
This allows us to handle the man pages just as a generic nested folder for install / uninstall rather than specifically handling the man case. I'd like to leave this like this for simplicity. It makes it obvious where a non.1 case would belong (arguablely brew should be .8)